CommunicationLayer/MessagingFramework/MessageDefines.h

```diff
@@ -4,11 +4,17 @@ namespace MessageDefines
 {
     enum Type
     {
-        KeyDriverControlTelemetry = 1,
-        DriverControlDetails = 2,
-        Faults = 3,
-        BatteryData = 4,
-        CmuData = 5,
+        KeyMotor = 1,
+        Motor0Details = 2,
+        Motor1Details = 3,
+        DriverControls = 4,
+        MotorFaults = 5,
+        BatteryFaults = 6,
+        Battery = 7,
+        Cmu = 8,
+        Mppt = 9,
+        Lights = 10,
+        Other = 11
```

Comment:
Generally enum values are treated as constants as well and shoudl follow `CAP_CASE`